docs/DeveloperGuide.md

```diff
@@ -255,13 +255,15 @@ _{Explain here how the data archiving feature will be implemented}_
 
 ### Product scope
 
-**Target user profile**:
+**Target user profiles**:
```

Comment:
Would this be a target user profile instead? since we are targeting a single user persona who satisfies all following criteria?

Reply:
Thanks for the feedback, I will update this.